trlx/orchestrator/ppo_orchestrator.py

```diff
@@ -55,9 +59,18 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
         Takes `num_rollouts` prompts from `pipeline`, samples model and computes the
         KL againts a reference model. It then appends PPOElements to trainer's `store`
         """
+        logger.info("Collecting rollouts")
+        tbar = tqdm(
+            total=num_rollouts,
+            disable=not os.environ.get("RANK", 0) == "0",
+            position=0,
+            leave=False,
```

Comment:
Why `leave` has to be false here? seems like colab doesn't particularly like it https://stackoverflow.com/a/57072638

Reply:
I thought leaving the progress bars would be too verbose but they show up in the stdout logs anyway. Updated!